includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -686,95 +703,198 @@ public function stripe_modal_checkout_status( $status = 'enabled' ) {
+											<?php if ( $slug === $default_account ) { ?>
+												<span class="give-stripe-account-default give-stripe-account-badge">
+													<?php esc_html_e( 'Default Account', 'give' ); ?>
+												</span>
+											<?php } else { ?>
+												<span class="give-stripe-account-default">
+													<a
+														data-account="<?php echo $slug; ?>"
+														data-url="<?php echo give_stripe_get_admin_settings_page_url(); ?>"
+														class="give-stripe-account-set-default" href="#"
+													><?php esc_html_e( 'Set as Default', 'give' ); ?></a>
+												</span>
+											<?php } ?>
+
+											</span>
+										</div>
+										<div class="give-stripe-account-actions">
+											<span class="give-stripe-account-type-description give-field-description"><?php _e( 'Connection Method:', 'give' ); ?></span>
+											<span class="give-stripe-account-type-method"><?php echo give_stripe_connection_type_names( $details['type'] ); ?></span>
+											<?php
+											if (
+												$slug !== $default_account ||
+												(
+													is_array( $stripe_accounts ) &&
+													count( $stripe_accounts ) === 1
+												)
+											) {
+												?>
+												<span class="give-stripe-account-disconnect">
+													<a
+														class="give-stripe-disconnect-account-btn"
+														href="<?php echo $disconnect_url; ?>"
+														data-disconnect-message="<?php echo $disconnect_message; ?>"
+													>
+																								<?php esc_html_e( 'Disconnect', 'give' ); ?>
+													</a>
+												</span>
+											<?php } ?>
+										</div>
+									</div>
+									<?php
+								}
+							} else {
+								?>
+								<div class="give-stripe-account-manager-list-item">
+									<span><?php esc_html_e( 'No Stripe Accounts found.', 'give' ); ?></span>
+								</div>
+							<?php } ?>
+						</div>
+						<div class="give-stripe-account-manager-add-section">
+							<h3><?php esc_html_e( 'Add New Stripe Account', 'give' ); ?></h3>
+							<div class="give-stripe-add-account-errors"></div>
+							<table class="form-table give-setting-tab-body give-setting-tab-body-gateways">
+								<tbody>
+								<?php
+								if ( give_stripe_is_premium_active() ) {
+									/**
+									 * This action hook will be used to load Manual API fields for premium addon.
+									 *
+									 * @param array $stripe_accounts All Stripe accounts.
+									 *
+									 * @since 2.7.0
+									 */
+									do_action( 'give_stripe_premium_manual_api_fields', $stripe_accounts );
+								}
+								?>
+								<tr valign="top" class="give-stripe-account-type-connect">
+									<th scope="row" class="titledesc">
+										<label for="stripe_connect_button">
+											<?php esc_html_e( 'Stripe Connection', 'give' ); ?>
+										</label>
+									</th>
+									<td class="give-forminp">
+										<?php echo give_stripe_connect_button(); ?>
```

Comment:
@mehul0810 I noticed that `$connected` is an unused variable in `give_stripe_connect_button` function. Can we remove this?